src/overlays/actors/ovl_En_Weather_Tag/z_en_weather_tag.c

```diff
@@ -177,7 +177,7 @@ u8 WeatherTag_CheckRestoreWeather(EnWeatherTag* this, GlobalContext* globalCtx,
 
     if ((WEATHER_TAG_RANGE100(this->actor.params) + 100.0f) < Actor_WorldDistXZToActor(&player->actor, &this->actor)) {
         if (globalCtx->envCtx.unk_1E != 0 || !D_8011FB3C ||
```

Comment:
```suggestion
        if ((globalCtx->envCtx.unk_1E != 0) || !(D_8011FB3C) ||
```

Reply:
definitely dont agree with parens around that bool.
But in general, we dont have set rules regarding parens. While I personally agree and like them used more often than not, we havent been enforcing anything specific and leave it to the discretion of the decompiler